resources/jolokia-access.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If it's basically always `jolokia-access.xml` and we don't think other folks will get confused then I'm good as-is.